### PR TITLE
Unify styles and add the nav bar for results.

### PR DESCRIPTION
**Pull request by hRobert12**
This pr will add the styles.js file, allowing us to put common styles in the file and then using it elsewhere in code. This pr also aims to add the nav bar to other screens besides the landing screen.

### CODE REVIEW

File: src/assets/styles.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { StyleSheet } from 'react-native';
+import colors from './colors';
+import metrics from '../themes/metrics.js'
+
+export const standardMargin = 16;
+export const standardPadding = 16;
+export const standardFontSize = 14;
+
+export const standardStyle = StyleSheet.create({
+    standardStyle: {
```
Comment (AndyCErnst) [inline src/assets/styles.js]: Change this to something more meaningful. It looks like it's only used in large text so call it largeText.

File: src/assets/styles.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        flex: 1,
+        justifyContent: 'space-between',
+    },
+    container: {
+        marginBottom: metrics.navbarMargin,
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: colors.transparent,
```
Comment (AndyCErnst) [inline src/assets/styles.js]: the default background color is transparent so you shouldn't need to specify it... unless it's overriding something

File: src/components/LargeText.js [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
-const styles = StyleSheet.create({
-  text: {
-    color: colors.darktext,
-    fontSize: 14
-  }
-});
+// Bad object: {
+//   "standardStyle": 23,
+//   "container": 24
+// }
```
Comment (AndyCErnst) [inline src/components/LargeText.js]: dead code

File: src/routes/results.js [hunk 4]
```diff
@@ -1,32 +1,34 @@
...
-            <Image source={images.recycleSymbol}/>
-            <Text>In recycling</Text>
-          </View>
-          <Text>Empty soda and bevrage cans should be rinsed first, then put in recycling</Text>
-        </FullWidthContainer>
-        <Button
-          title='Go To Landing'
-          onPress={() => this.props.navigation.navigate('Landing')}
-        />
+      <View style = {standardStyle.wrapper}>
```
Comment (AndyCErnst) [inline src/routes/results.js:15]: The extra spaces in this file: `style = {` etc make it slightly harder to read. I should look into an autoformatter so robots can help standardize.

File: src/routes/results.js [hunk 5]
```diff
@@ -1,32 +1,34 @@
...
+        <View style={standardStyle.container}>
+          <LargeText style={styles.headerUpper} text = "How to recycle"/>
+          <LargeText style={styles.headerLower} text = "Beverage cans"/>
+          <FullWidthContainer flex={containerFlex}>
+            <View style={{flexDirection: 'row', alignItems: 'center'}}>
+              <Image source={images.recycleSymbol} />
+              <LargeText text = "In recycling" style={{flex: 2}}/>
```
Comment (AndyCErnst) [inline src/routes/results.js]: I know this is just the quick code I sent you, but let's pull out these inline styles into a style sheet and try something like this:
```
          <View style={styles.result}>
              <Image source={images.recycleSymbol} style={styles.icon}/>
              <LargeText text = "In recycling" style={styles.instructionHeader}/>
            </View>
```
```
instruction: {flexDirection: 'row', alignItems: 'center'},
instructionHeader: {flex: 2},
icon: {width: 45} // I think this is their size, double check
```

File: src/routes/results.js [hunk 6]
```diff
@@ -1,32 +1,34 @@
...
+            </View>
+            <LargeText text = "Empty soda and bevrage cans should be rinsed first, then put in recycling"/>
```
Comment (AndyCErnst) [inline src/routes/results.js]: typo in "beverage" but this is placeholder content so, eh

File: package.json [hunk 7]
```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "expo": "^27.0.1",
     "react": "16.3.1",
+    "react-devtools": "^3.2.3",
```
Comment (AndyCErnst) [inline package.json:24]: from the react-devtools readme https://github.com/facebook/react-devtools/blob/master/packages/react-devtools/README.md, it doesn't sound like you need to add it to the project. They suggest installing it globally (that's the `-g` in the npm command). They run it from the command-line... we'll have to see if that works

**Review by AndyCErnst — CHANGES_REQUESTED** [review]
Thanks for pulling out the style into another file. In a big project we might want to have different style files for layout, text, buttons, inputs, etc but for now we can be just be sure to name them well in the file

File: src/routes/results.js (hunk 4 above)
Comment (hRobert12) [reply src/routes/results.js:15]: Interesting, I always thought that having spaces around the equals made it easier to read.

Comment (AndyCErnst) [reply src/routes/results.js:15]: ah, in regular code, yes. I realize now that the style for jsx must be inherited from html where attributes are written without spaces `<div class="classname">`

Comment (AndyCErnst) [reply src/routes/results.js:15]: So, spaces in javascript, no spaces in jsx (even though the jsx is inside of javascript) :D

**Comment by SethDonohue** [comment]
Did you pull the most recent code from master to this branch? my changes that were merged yesterday may affect what you have done!

**Comment by AndyCErnst** [comment]
oddly, this PR is showing one of Seth's files that's already in master. Maybe that will fix itself if you use the resolve conflicts button on github.

**Review by AndyCErnst — APPROVED** [review]

**Pull request merged**